styles.css

```diff
@@ -1,6 +1,108 @@
-body{
-    color: #bae1ff;
+:root {
+    --primary-color: #370140;
+    --secondary-color: #bae1ff;
 }
-h1{
-    color:#6b37c1;
+
+/* Create and style grid container for the banner */
+
+.center {
+    margin-left: auto;
+    margin-right: auto;
+}
+
+/* .width-sm {
+    max-width: 20rem;
+} */
+
+.width-lg {
+    max-width: 40rem;
+}
+.width-xl {
+    max-width: 60rem;
+}
+
+#landing {
+    background-color: var(--primary-color);
+}
+
+.grid-container {
+    /* border: solid black 2px; */
+    display: grid;
+    grid-template-columns: repeat(3, minmax(10rem, 1fr));
+    grid-template-rows: repeat(7, 5rem);
+    grid-template-areas: 
+        "about about ."
+        "about about casestudies"
+        "services ved casestudies"
+        "services ved casestudies"
+        "services ved . "
+        "services contact contact"
+        ;
+    column-gap: 2rem;
```

Comment:
gaps are precise, how do these set sizes look with all screens? ⭐

Reply:
I'll work on that today actually and will let you know!

Reply:
That't great, looking forward to seeing it 😄